content/docs/dvclive/usage.md

```diff
@@ -0,0 +1,160 @@
+# Usage
+
+We will use sample [MNIST classification](http://yann.lecun.com/exdb/mnist/)
+training code in order to see how one can introduce `dvclive` into the workflow.
+In order to run the example,
+[keras](https://keras.io/about/#installation-amp-compatibility) is required.
+
+The training code (`train.py` file):
+
+```python
```

Comment:
I should update the example repo I have with this example. Keras really makes for a clean, minimal example.

Reply:
Yeah, but on the other hand we need to prepare `Callback`, while in `torch` everything nicely goes into the training loop